src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package org.micks.champmaker.championships;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.micks.champmaker.EntityNotFoundException;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class ChampionshipServiceTest {
+
```

Comment:
(n2h) You can delete first blank line of the class.

Reply:
I like when field or method has empty line before 🙂